static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -170,6 +173,8 @@ class GroupDetails extends Component<Props, State> {
         event
       ),
       ref_fallback,
+      num_participants: group?.participants.length ?? 0,
+      num_viewers: group?.seenBy.filter(user => user.id !== activeUser.id).length ?? 0,
```

Comment:
@malwilley can you add this to `getAnalyticsDataForGroup`?

Reply:
also we shouldn't filter out the user who saw this. Just unnecessary logic.

Reply:
Yeah can remove that, just note that this num_viewers number will differ from what we show in the UI since we do filter there

Reply:
@malwilley oh TIL, yea let's have it match that number. My bad.